test/e2e/workstatus-creation.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
```

Comment:
Maybe we can remove this file since it is covered by workstatus-crud.sh?

Reply:
done.